lib/utils.js

```diff
@@ -1,7 +1,75 @@
+const logger = require('./logger')
+const SierraItem = require('./sierra-models/item')
+const nyplCoreItemTypes = require('@nypl/nypl-core-objects')('by-catalog-item-type')
+const nyplCoreLocations = require('@nypl/nypl-core-objects')('by-sierra-location')
+
 const isValidResponse = (resp) => {
   return resp && resp.data
 }
 
+/**
+* Given an array of items, resolves a new array of items containing only
+* those items that *may* be Research based on Item Type
+*/
+const prefilterItems = async function (items) {
+  const filteredItems = items.filter((item) => {
+    if (item.nyplSource !== 'sierra-nypl') return true
```

Comment:
can you put comments describing whats happening for each of these checks? ie like 
`// if its not a sierra item, its a partner item, hence research `
(like you have for the bibprefilter)